### PR TITLE
Show evaluation time in the evaluations overview

### PR DESCRIPTION
**Pull request by dasJ**
image

### CODE REVIEW

File: src/lib/Hydra/Controller/Jobset.pm [hunk 1]
```diff
@@ -45,6 +45,11 @@ sub jobset_GET {
 
     $c->stash->{latestEval} = $c->stash->{jobset}->jobsetevals->search({ hasnewbuilds => 1 }, { rows => 1, order_by => ["id desc"] })->single;
 
+    $c->stash->{duration2human} = sub {
+        my ($duration) = @_;
+        return Time::Seconds->new($duration)->pretty();
+    };
+
```
Comment (grahamc) [inline src/lib/Hydra/Controller/Jobset.pm:52]: I think this should probably move to TT.pm:

```diff
--- a/src/lib/Hydra/View/TT.pm
+++ b/src/lib/Hydra/View/TT.pm
@@ -9,7 +9,7 @@ __PACKAGE__->config(
     ENCODING => 'utf-8',
     PRE_CHOMP => 1,
     POST_CHOMP => 1,
-    expose_methods => [qw/buildLogExists buildStepLogExists jobExists stripSSHUser/]);
+    expose_methods => [qw/buildLogExists buildStepLogExists jobExists stripSSHUser relativeDuration/]);
 
 sub buildLogExists {
     my ($self, $c, $build) = @_;
@@ -34,6 +34,11 @@ sub stripSSHUser {
     }
 }
 
+sub relativeDuration {
+    my ($self, $c, $seconds) = @_;
+    return Time::Seconds->new($seconds)->pretty();
+}
+
```

**Review by grahamc — CHANGES_REQUESTED** [review]
Looks pretty good! I think the function should probably live somewhere else, to prevent confusion if someone tries to use it elsewhere. I'm not totally sure if this is the best answer, but I'd be happy enough with it.

**Comment by grahamc** [comment]
I applied my change over in https://github.com/NixOS/hydra/compare/master...DeterminateSystems:pr-887-review  -- I propose we squash these commits if you're up for it.

**Comment by dasJ** [comment]
Like this?

**Review by grahamc — APPROVED** [review]
lgtm, thanks! just waiting for CI to be green

**Pull request merged**